### ISSUE TITLE
Term Request: Corneal lattice lines

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11744]
**Preferred term label:**
Corneal lattice lines

**Synonyms**
* Branching refractile corneal lines
* Corneal stromal amyloid lattice lines
* Refractile branching corneal lines

**Definition (free text, please give PubMed ID)**
Fine or ropy, branching, refractile lines crisscrossing within the corneal stroma to produce a lattice network. These lines are composed of extracellular amyloid fibrils that demonstrate birefringence and green dichroism under polarized light after Congo red staining (PMID: 38359414).

**Parent term (use hpo.jax.org/app)**
Opacification of the corneal stroma (HP:0007759)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
* Classic lattice corneal dystrophy (LCD1) (ORPHA:98964, OMIM:122200)
* Lattice corneal dystrophy variants (LCD variants) (OMIM:608471)
* Granular corneal dystrophy, type 2 (GCD2) (ORPHA:98963, OMIM:607541)
* Amyloidosis, Finnish type / Meretoja syndrome (ORPHA:85448, OMIM:105120)

**Your nano-attribution (ORCID)**
0000-0001-9042-0508
